README.md

```diff
@@ -44,16 +45,17 @@ See [INSTALL.md](/INSTALL.md) for detailed installation instructions.
 
 ### Versioning
 
-`client-go` follows [semver](http://semver.org/). We will not make
-backwards-incompatible changes without incrementing the major version number. A
-change is backwards-incompatible either if it *i)* changes the public interfaces
+- For each `v1.x.y` Kubernetes release, the major version (first digit)
+would remain `0`.
+
+- Changes that add features in a backwards-compatible way will
```

Comment:
This paragraph doesn't make a lot of sense any more, I would drop it.

Reply:
Dropped